.github/workflows/test-template.yml

```diff
@@ -286,6 +287,11 @@ jobs:
         flavor: ['js_pg', 'js_neon', 'js_libsql', 'js_planetscale']
         shard: ['1/6', '2/6', '3/6', '4/6', '5/6', '6/6']
         node: [20] #[16, 18, 20]
+        previewFeatures: ['driverAdapters', 'driverAdapters,relationJoins']
+        exclude:
+          # Exclude relationJoins tests with WASM because the WASM engine does not include the necessary changes yet.
+          - engine-type: 'wasm'
+            previewFeatures: 'driverAdapters,relationJoins'
```

Comment:
Could you please create a small ticket for us to get back to this later (once wasm engine publish/update is streamlined).

Reply:
https://github.com/prisma/team-orm/issues/697